medical/index.css

```diff
@@ -78,33 +78,33 @@ margin-right: 60px;
  background-color: #1BA0AB;
  color: white ;
  border-radius: 4px;
- font-family: 'Montserrat', sans-serif !important;
+ font-family: 'Montserrat', sans-serif ;
 }
 .top__button--active button svg{
     margin-left: 5px;
 }
-.welcome{
+.par{
     background-color: #FFF5F0;
     padding-left: 10%;
     padding-right: 5%;
     margin: 0;
     height: 70vh;
 }
-.welcome__smalltitle{
-    font-style: normal;
+.par__secondtitle{

```

Comment:
can you try to use the same naming convention, above you used the kebab case you should also use it here like this `.par__second-title`